src/App.js

```diff
@@ -0,0 +1,12 @@
+import './App.css';
+import Calculator from './components/Calculator';
+
+function App() {
```

Comment:
- [ ] Please the App.js component should be class-based and not the function as required in the project 🙃 🙃 

eg: 

```javascript
import React, { Component } from 'react';
import Calculator from './components/Calculator'; 
import './App.css';

class App extends Component {
  render() {

    return {
      <Calculator />
    };
  }
}
```

Reply:
Now I did it with classes, but it seems not to work with linters.
 It says : Component should be written as a pure function

Reply:
Hello @maaloum . You might have to add this line at the top of the component file ```/* eslint-disable react/prefer-stateless-function */```